Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -128,6 +139,37 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+/// Returns an identity matrix or a batch of matrices.
+///
+/// - Parameters:
+///   - rowCount: The number of rows in each batch matrix.
+///   - columnCount: The number of columns in each batch matrix.
+///   - batchShape: The leading batch dimensions of the returned tensor.
+/// TODO: Make columnCount and batchShape optional.
```

Comment:
Please remove obsolete comment:
```suggestion
```

Aside: `//` should be used for `TODO` comments. `///` is reversed for API documentation comments.